cable_follower:
  type: mars_control/CableFollower
  arm_id: anan
  is_vertical: false 
  p_gain: -0.05
  gelsight_topic: /combined_arms/an_an/gelsight/pose
  output_topic: /cable_data_collection
